client/src/routes/automation/BlocklyComponent.svelte

```diff
@@ -39,7 +72,7 @@
 
 <style scoped>
     #blocklyDiv {
-        height: 600px;
+        height: 700px;
```

Comment:
Is there a way to use relative values instead ?

Reply:
In the next version, when I will add my tailwind implementation